### ISSUE TITLE
Copier plugin

### ISSUE THREAD

**Issue by Magnificia** [Vencord/plugin-requests#829]
### Plugin Description

I want the Copier plugin to be added cause it is easier then View Raw and quicker 
![image](https://github.com/Vencord/plugin-requests/assets/62173194/1541d5ae-5958-47c1-ba4a-b1aee5ce7ee5)

### Existing Plugin for other mod

https://vencord.dev/plugins/BetterRoleContext      https://vencord.dev/plugins/ViewRaw

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I checked every box, despite not reading any
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Inbestigator** [comment]
You can just rightclick the view raw button to copy

**Comment by MrDiamondDog** [comment]
> You can just rightclick the view raw button to copy

> it is easier than View Raw

Also dont you still have to copy it from the view raw dialog cause right clicking the view raw button does nothing

**Comment by Inbestigator** [comment]
> > You can just rightclick the view raw button to copy
> 
> > it is easier than View Raw
> 
> Also dont you still have to copy it from the view raw dialog cause right clicking the view raw button does nothing

No, it just copies the raw message content, I use it all the time.
